packages/react/src/NavList/NavList.tsx

```diff
@@ -413,6 +415,104 @@ const Group: React.FC<NavListGroupProps> = ({title, children, sx: sxProp = defau
   )
 }
 
+export type NavListShowMoreItemProps = {
+  children: React.ReactNode
+  label?: string
+  pages?: number
+} & SxProp
```

Comment:
Feel free to just remove any `sx` stuff if you don't want to have to deal with toggling behind the flag!

```suggestion
}
```